plugins/GrapesJsBuilderBundle/Model/GrapesJsBuilderModel.php

```diff
@@ -70,7 +70,7 @@ public function addOrEditEntity(Email $email)
 
         $this->getRepository()->saveEntity($grapesJsBuilder);
 
-        $customHtml = ArrayHelper::getValue('customHtml', $this->requestStack->getCurrentRequest()->get('emailform'));
+        $customHtml = ArrayHelper::getValue('customHtml', $this->requestStack->getCurrentRequest()->get('emailform'), []);
```

Comment:
@luguenth the fix doesn't work because the `getValue()` function expects an array in the 2nd function argument, meaning that your fix (which adds a 3rd argument) unfortunately doesn't work in this case. The `get()` function on `getCurrentRequest()` also supports a default parameter, so the fix is to simply move the default value as can be seen below:

```suggestion
        $customHtml = ArrayHelper::getValue('customHtml', $this->requestStack->getCurrentRequest()->get('emailform', []));
```

Reply:
Ah I see :+1: Thanks!